lib/PuppeteerSharp/Browser.cs

```diff
@@ -66,6 +66,8 @@ public class Browser : IDisposable
 
         #region Private members
         internal readonly Dictionary<string, Target> TargetsMap;
+        internal readonly bool _setDefaultViewport;
```

Comment:
How about an internal readonly property?
I don’t like much the idea of accessing `_fields` from other classes.